.semaphore/semaphore.yml

```diff
@@ -51,6 +51,7 @@ blocks:
                 - .ci/validation.sh eclipse-static-analysis
                 - .ci/validation.sh eclipse-static-analysis-java11
                 - .ci/validation.sh java11-verify
+                - .ci/validation.sh verify-regexp-id
```

Comment:
@romani where else we need to add this check?

Reply:
It is enough